ui/pages/keychains/reveal-seed.js

```diff
@@ -138,7 +138,7 @@ const RevealSeedPage = () => {
           error={error}
           width={BLOCK_SIZES.FULL}
         />
-        {error && <HelpText error>{error}</HelpText>}
+        {error && <HelpText severity={SEVERITIES.DANGER}>{error}</HelpText>}
```

Comment:
Updating HelpText to show in error/danger using severities 

<img width="363" alt="Screenshot 2023-02-15 at 4 27 16 PM" src="https://user-images.githubusercontent.com/8112138/219226743-943bd360-820b-491f-b082-94fa337abd45.png">